theme/accessibility.less

```diff
@@ -9,9 +9,16 @@
           Accessibility
 *******************************/
 
-*:focus{
-    outline: 2px solid #4D90FE !important;
+*:focus {
+    outline: 1px solid transparent !important;
 }
```

Comment:
Is this necessary? Setting it to transparent?? Why not stick with whatever focus outline is set by the component itself.

Reply:
So the reason why is that on Android, when there is no outline defined, a Red outline appear.

There was a solution couple of year ago but it is not supported anymore. The fixed I've found is to put it to transparent. "none" was not working. [https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-tap-highlight-color](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-tap-highlight-color)